charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml

```diff
@@ -282,9 +282,9 @@ data:
         Kube_URL            https://kubernetes.default.svc.cluster.local:443
         Buffer_Size         1M
         Labels              On
-        Annotations         Off
+        Annotations         On
         tls.verify          Off
-        K8S-Logging.Exclude On
+        K8S-Logging.Exclude Off
```

Comment:
Do you think it makes sense to change these settings in a second step, a few weeks later? There might still be plenty of pods with the exclude annotation.

Reply:
Regarding `K8S-Logging.Exclude` I think it should be done now, in this way logs for still annotated pods will be collected.
Regarding `Annotations` - why should it be enabled?

Reply:
> I think it should be done now, in this way logs for still annotated pods will be collected.

Can you explain why you want to do this? Tbh, I don't know the details but I was thinking that it might not be a good idea to suddenly record logs of pods which were previously excluded due to `"fluentbit.io/exclude": "true"` and rather give it some time to reduce the odds such pods still exist in the control plane.

Reply:
The new version of fluent-bit will ignore those pods. And if a shoot purpose is changed from testing to production, this will prevent the bug of not removing the annotations. The new version of fluent-bit plugin watch Cluster resourses from witch deduct the shoot purpose.

Reply:
Okay got it, thanks for the explanation.